src/components/ElvaYBPractice2/styles/MainComponent.styled.ts

```diff
@@ -0,0 +1,258 @@
+
+export const MainComponent = styled.div`
+z-index: 2;
+border-radius: 15px;
+width: max-content;
+height: auto;
+display: flex;
+padding-top: 30px;
+background-color: #ffffff;
+align-items: center;
+justify-content: center;
+flex-direction: column;
+box-shadow: 0px 3px 10px rgb(0 0 0 / 0.2);
+@media only screen and (max-width: 600px) {
+  margin-top: 180px;
+  height: 60vh;
+  width: 55vw;
+
+  
+`
+
+export const ToggleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-left: 120px;
+
+  width: 80%;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+    margin-left: 40px;
+    top: -10px;
+  }
+`
+export const SlidercontainerDesktop = styled.div`
+  width: 60%;
+  align-items: center;
+  display: block;
+  flex-direction: row;
+
+  @media only screen and (max-width: 600px) {
+    width: 85%;
+    top: -85px;
+    display: none;
+  }
+`
+export const SlidercontainerMobile = styled.div`
+  width: 60%;
+  align-items: center;
+  flex-direction: row;
+  display: none;
+
+  @media only screen and (max-width: 600px) {
+    width: 85%;
+    display: block;
+    margin-left: 15px;
+    padding-top: 30px;
+  }
+`
+
+export const MainBottomComponent = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  height: auto;
+  box-shadow: 0 -1px 0 #d3d3d3;
+  margin-top: 30px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+
+  @media only screen and (max-width: 600px) {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 30px;
+  }
+`
+export const PageViewPriceContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+
+  @media only screen and (max-width: 600px) {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0px;
+
+    height: 50%;
+  }
+`
+export const PageViewContainer = styled.div`
+  width: 50%;
+  color: hsl(225, 20%, 60%);
+  text-align: center;
+  margin-left: 20px;
+  font-size: 15px;
+  font-weight: 500;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+    margin-left: 0px;
+
+    font-size: 14px;
+  }
+`
+export const PriceContainer = styled.div`
+  width: auto;
+  text-align: right;
+  font-weight: bold;
+  color: hsl(227, 35%, 25%);
+  font-size: 30px;
+  margin-top: -13px;
+
+  @media only screen and (max-width: 600px) {
+  }
+`
+export const PriceMonthContainer = styled.div`
+  width: min-content;
+  display: flex;
+  flex-direction: row;
+
+  margin-left: auto;
+  margin-right: auto;
+  @media only screen and (max-width: 600px) {
+    padding-top: 20px;
+    height: min-content;
+  }
+`
+export const MonthContainer = styled.div`
+  width: min-content;
+  text-align: center;
+  color: hsl(225, 20%, 60%);
+  margin-left: 0px;
+
+  @media only screen and (max-width: 600px) {
+  }
+`
+export const StartTrialButton = styled.button`
+  background-color: hsl(227, 35%, 25%);
+  border-radius: 20px;
+  width: 65%;
+  height: 35%;
```

Comment:
can consider setting height to auto. Even the width too.

Reply:
Resolved in commit d2f947f.